docs/jwst/charge_migration/description.rst

```diff
@@ -0,0 +1,52 @@
+Description
+===========
+
+:Class: `jwst.charge_migration.ChargeMigrationStep`
+:Alias: charge_migration
+
+
+Overview
+--------
+This step corrects for an artifact seen in undersampled NIRISS images that may depress flux 
+in resampled images. The artifact is seen in dithered images where the star is centered in 
+a pixel. When the peak pixels of such stars approach the saturation level, they suffer from 
+significant :ref:`charge migration <charge_migration>`:
+the spilling of charge from a saturated pixel into its neighboring pixels. This charge migration 
+causes group-to-group differences to decrease significantly once the signal level is greater than 
+~25,000 ADU. As a result, the last several groups of these ramps get flagged by the :ref:`jump <jump_step>`
+step.  The smaller number of groups used for these pixels in the :ref:`ramp_fitting <ramp_fitting_step>`
+step results in them having  larger read noise variances, which in turn leads to lower weights used
+during resampling. This ultimately leads to a lower than normal flux for the star in resampled images.
+
+Once a group in a ramp has been flagged as affected by charge migration, all subsequent 
+groups in the ramp are also flagged. By flagging these groups, they are not used in the 
+computation of slopes in the :ref:`ramp_fitting <ramp_fitting_step>` step. However, as described 
+in the algorithm section below, they _are_ used in the calculation of the variance of the slope 
+due to readnoise.
+
+Input details
+-------------
+The input data must have been processed through the :ref:`jump <jump_step>` step, so the input must be in the
```

Comment:
This statement is now out of date, because this step now comes *before* the jump step. Need to update.